gnd/src/main/java/com/google/android/gnd/repository/OfflineAreaRepository.java

```diff
@@ -109,19 +148,17 @@ public Completable addAreaAndEnqueue(LatLngBounds bounds) {
 
   public Flowable<ImmutableSet<Tile>> getIntersectingDownloadedTilesOnceAndStream(
       OfflineArea offlineArea) {
-    File jsonSource;
-
-    try {
-      jsonSource = fileUtil.getFileFromRawResource(R.raw.gnd_geojson, Config.GEO_JSON);
-    } catch (IOException e) {
-      return Flowable.error(e);
-    }
-
-    ImmutableList<Tile> tiles =
-        geoJsonParser.intersectingTiles(offlineArea.getBounds(), jsonSource);
-
-    return getDownloadedTilesOnceAndStream()
-        .map(ts -> stream(tiles).filter(tiles::contains).collect(toImmutableSet()));
+    return projectRepository
+        .getActiveProjectOnceAndStream()
+        .compose(Loadable::values)
+        .map(Project::getOfflineBaseMapSources)
+        .map(this::downloadOfflineBaseMapSource)
+        .map(json -> geoJsonParser.intersectingTiles(offlineArea.getBounds(), json))
+        .flatMap(
+            tiles ->
+                getDownloadedTilesOnceAndStream()
+                    .map(ts -> stream(ts).filter(tiles::contains).collect(toImmutableSet())))
+        .onErrorReturn(throwable -> ImmutableSet.of());
```

Comment:
Is the error non-fatal? If not, then please consider adding `Timber.e()` so that it eventually ends up in error/crash reporting.

Reply:
It's not fatal in this case--added a brief explanation. 80e58f8